API/Commands/AddSkill.cs

```diff
@@ -0,0 +1,158 @@
+﻿using PathOfTerraria.Content.Items.Gear;
+using PathOfTerraria.Core.Systems.SkillSystem;
+using System.Linq;
+using Terraria.ModLoader.Core;
+
+namespace PathOfTerraria.API.Commands;
+
+[Autoload]
+public class AddSkill : ModCommand {
+	public override string Command => "addskill";
+
+	public override CommandType Type => CommandType.Chat;
+
+	public override string Usage => "[c/ff6a00:Usage: /addskill <skillName> <skillSlot> <duration> <timer> <maxCooldown> <coolDown> <manaCost> <weaponType> <level>]";
```

Comment:
Seems a bit of a nightmare, glad there are defaults lol

Reply:
I wanted to make sure we could easily debug every single part of the skill. This command sucks, no doubt about it, but I want to make sure it's versatile.